docs/kyma/05-03-overrides.md

```diff
@@ -85,10 +85,10 @@ While installing Kyma, provide the file path using the `-o` flag. Once the insta
 
 For overrides that the system should keep in Secrets, just define a Secret object instead of a ConfigMap with the same key and a base64-encoded value. Be sure to label the Secret.
 
-If you add the overrides after installation, trigger the update process using this command:
+If you add the overrides after installation, trigger the update process using Kyma CLI providing the same version of the installed Kyma:
```

Comment:
```suggestion
If you add the overrides after installation, trigger the update process using Kyma CLI. Provide the same version of the installed Kyma:
```

Reply:
It's just a suggestion as I guess it's more readable this way. (or `using Kyma CLI and providing`...)